docs/Makefile

```diff
@@ -15,10 +15,10 @@ apidoc-watch-build:
 next-dev-install:
 	cd next; yarn install
 
-next-watch-build:
+next-watch-build dev:
```

Comment:
What's goin' on here?

Reply:
this just adds an alias for the command, so you can just do `make dev` or `make next-watch-build`